kubernetes-event-exporter.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: kubernetes-event-exporter
+  version: 1.5
+  epoch: 0
+  description: Export Kubernetes events to multiple destinations with routing and filtering
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/resmoio/kubernetes-event-exporter
+      tag: v${{package.version}}
+      expected-commit: b892cb20dfbc6bcb8ca7b3963c8a6446161431ef
+
+  - runs: |
+      # Handle CVE-2022-41723 CVE-2023-39325 and CVE-2023-3978
+      go get golang.org/x/net@v0.17.0
+      go mod tidy
+      VERSION=${{package.version}}}
+      PKG=github.com/resmoio/kubernetes-event-exporter/pkg
+      CGO_ENABLED=0 GOOS=linux GO11MODULE=on go build -ldflags="-s -w -X ${PKG}/version.Version=${VERSION}" -a -o /main .
```

Comment:
Could we use the `go/build` pipeline instead?

Reply:
yep forgot to do that, thanks for reminding me!